pom.xml

```diff
@@ -1733,7 +1733,7 @@
               <configuration>
                 <rules>
                   <requireMavenVersion>
-                    <version>${maven.version}</version>
+                    <version>[3.3,)</version>
```

Comment:
${maven.version} and [${maven.version},) are actually the same think for an enforcer perspective. So, if we don't go to [3.3,), then I will close this PR.